torch/distributed/_functional_collectives.py

```diff
@@ -1046,7 +1046,7 @@ def _reduce_scatter_tensor_coalesced_native_meta(
 def all_gather_tensor_inplace(
     output_tensor: torch.Tensor,
     input_tensor: torch.Tensor,
-    group,  # TODO add a type,
+    group=None,  # TODO add a type
```

Comment:
I think this fix looks good, thanks!

Would you like to add a unit test similar to https://github.com/pytorch/pytorch/blob/46624359c3ccea12c288068cfa7df2c0b849a11b/test/distributed/test_inductor_collectives.py#L378-L401, but not passing in the `pg` argument?